language/builtin.hlb

```diff
@@ -218,6 +218,17 @@ option::run secret(string localPath, string mountPoint)
 # @return an option to mount an additional filesystem.
 option::run mount(fs input, string mountPoint) binds (fs target)
 
+# Sets a debugger breakpoint just before the "run" command.
+#
+# @param command optional command to invoke for the debugger "exec" command,
+# followed by optional arguments
+# If no value is given, it will execute /bin/sh.
+# If exactly one value is given, the specified command will be invoked.
+# If more than one value is given, values after the first one are treated as
+# arguments for the command.
+# @return an option to set a breakpoint
+option::run breakpoint(variadic string command)
```

Comment:
this implies `breakpoint` without arguments will work,  but I think we wanted `breakpoint(string command, variadic string args)` so as least one argument is required.

Reply:
I was thinking it should be okay to specify `breakpoint` without args and default to `/bin/sh` for an exec. That would keep things consistent with the `breakpoint` statement. Now that we've decided we won't use the `run` command as an exec default, I don't think there's ambiguity in what gets run. But I'm fine with requiring at least one arg if you prefer.

Reply:
okay, works for me 👍